runs/great3/old/mysimparams.py

```diff
@@ -1,177 +0,0 @@
-"""
```

Comment:
Maybe let's keep these files ?

All the rest is fine (I checked only the "package" changes), OK for merge!